packages/utils/package.json

```diff
@@ -16,66 +16,55 @@
     "url": "https://github.com/boomerang-io/webapp-packages/issues"
```

Comment:
@LucasGrimauth from here down can be ignored - this me adding stuff to the utils package before realizing before branching off.